ktor-utils/common/src/io/ktor/util/internal/LockFreeLinkedList.kt

```diff
@@ -11,8 +11,6 @@ package io.ktor.util.internal
  */
 
 import kotlinx.atomicfu.*
-import kotlin.jvm.*
```

Comment:
Can you revert it please? The build fails on native without it.

Reply:
Solved.